packages/headless/src/features/instant-results/instant-results-actions.ts

```diff
@@ -73,7 +74,7 @@ export interface FetchInstantResultsActionCreatorPayload {
    */
   maxResultsPerQuery: number;
   /**
-   * The maximum items to be stored in the instant result list for each query.
+   * Number in milliseconds that cached results will be valid for. Defaults to 1 minute. Set to 0 so that results never expire.
```

Comment:
Headless has its own default of 1 minute or it assumes SearchAPI's default is of 1 minute?